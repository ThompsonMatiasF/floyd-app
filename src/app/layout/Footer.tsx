import React from 'react'

const Footer = () => (
  <div className="flex flex-col md:flex-row text-center md:text-base justify-between text-xs pt-4 pb-2 px-8 md:px-0">
    <span>Developer: <span className="font-bold">Matias Thompson</span></span>
  </div>
)

export default Footer
