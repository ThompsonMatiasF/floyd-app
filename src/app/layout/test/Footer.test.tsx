import { render } from '@testing-library/react'
import Footer from '../Footer'

describe('Footer.tsx', () => {
  it('should match snapshot', () => {
    const container = render(<Footer />).container

    expect(container).toMatchSnapshot()
  })
})
