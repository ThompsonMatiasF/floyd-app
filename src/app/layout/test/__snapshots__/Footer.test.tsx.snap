// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.tsx should match snapshot 1`] = `
<div>
  <div
    class="flex flex-col md:flex-row text-center md:text-base justify-between text-xs pt-4 pb-2 px-8 md:px-0"
  >
    <span>
      Developer: 
      <span
        class="font-bold"
      >
        Matias Thompson
      </span>
    </span>
  </div>
</div>
`;
